common/src/main/cpp/classes/openxr_fb_passthrough_geometry.cpp

```diff
@@ -91,6 +123,37 @@ void OpenXRFbPassthroughGeometry::destroy_passthrough_geometry() {
 		OpenXRFbPassthroughExtensionWrapper::get_singleton()->destroy_geometry_instance(geometry_instance);
 		geometry_instance = XR_NULL_HANDLE;
 	}
+
+	if (opaque_mesh != nullptr) {
+		remove_child(opaque_mesh);
+		opaque_mesh->queue_free();
+		opaque_mesh = nullptr;
+	}
+}
+
+void OpenXRFbPassthroughGeometry::instatiate_opaque_mesh() {
+	opaque_mesh = memnew(MeshInstance3D);
```

Comment:
Should add an error condition if `opaque_mesh` is not null, and one if `!mesh.is_valid()`.

Reply:
Added!